ios/FluentUI.Demo/FluentUI.Demo/Demos/SideTabBarDemoController.swift

```diff
@@ -227,7 +228,7 @@ extension SideTabBarDemoController: SideTabBarDelegate {
         contentViewController?.present(alert, animated: true)
     }
 
-    func sideTabBar(_ sideTabBar: SideTabBar, didActivate avatarView: AvatarView) {
+    func sideTabBar(_ sideTabBar: SideTabBar, didActivate avatarView: MSFAvatar) {
```

Comment:
The naming standard in FluentUI is something that has been changed back and forth a couple of times.
The reason we decided to prefix types with MSF in Vnext was because of the Button struct in SwiftUI that clashes with the FluentUI Button.
This is the reason why the FluentUI SwiftUI button is MSFButtonView and the UIKit wrapper is MSFButton.

We can take the discussion of standards offline and update the entire code base with the decision we make on a separate work item.